### PR TITLE
Add gym and pygame to tensorflow image

### PR DESCRIPTION
**Pull request by DavidFair**
Adds PyGame and Gym to the Tensorflow image for an upcoming SciML course.

As these are relatively small deps, I'll add them to the tensorflow image by default rather than create a new one

### CODE REVIEW

File: jupyter-tensorflow-notebook/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,6 @@
...
-    nvidia-headless-515 \
-    nvidia-utils-515 \
-    libnvidia-gl-515-server \
-    libopengl0 \
-    && apt-get install --yes \
-    curl \
-    wget \
-    patch  \
-    nvidia-cuda-toolkit \
-    && rm -rf /var/lib/apt/lists/*
```
Comment (anish-mudaraddi) [inline jupyter-tensorflow-notebook/Dockerfile:18]: I can't see any of the upstream docker containers install the `nvidia` or `opengl` packages - are they not needed anymore?

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]

File: jupyter-tensorflow-notebook/Dockerfile (hunk 1 above)
Comment (anish-mudaraddi) [reply jupyter-tensorflow-notebook/Dockerfile:18]: @meoflynn informs me that these are not needed

**Review by anish-mudaraddi — APPROVED** [review]

**Review by meoflynn — CHANGES_REQUESTED** [review]
Tensorflow doesn't seem to be picking up GPU devices, there may be some missing dependencies that Tensorflow specifically requires

**Review by meoflynn — APPROVED** [review]
Issues with tensorflow dependencies to be fixed in a future PR

**Pull request merged**